src/analysis/seminorm.lean

```diff
@@ -708,6 +708,71 @@ section normed_field
 variables [normed_field 𝕜] [add_comm_group E] [module 𝕜 E] (p : seminorm 𝕜 E) {A B : set E}
   {a : 𝕜} {r : ℝ} {x : E}
 
+lemma l (x : E) (p q : seminorm 𝕜 E) : bdd_below (range (λ (u : E), p u + q (x - u))) :=
```

Comment:
This could use a slightly more descriptive name :wink: 
```suggestion
lemma bdd_below_range_add (x : E) (p q : seminorm 𝕜 E) : bdd_below (range (λ (u : E), p u + q (x - u))) :=
```